src/main/java/com/kainos/ea/backend/models/JobRole.java

```diff
@@ -21,10 +22,10 @@
     private String specification;
 
     @Column(name = "sharepointURL")
-    private String sharepointUrl;
+    private String sharepointUrl = "https://kainossoftwareltd.sharepoint.com/people/SitePages/Career-Lattice.aspx";
```

Comment:
Would it not make more sense for it to default to a null?

Reply:
We have discussed for the default URL to be a sharepoint URL, this is why I've set it like that, we can change it after another discussion if needed

Reply:
After a night's worth of sleep I agree with you that I should set it to NULL and handle the nulls in the browser